server/src/main/java/com/unosquare/carmigo/service/PlatformUserService.java

```diff
@@ -54,13 +54,14 @@ public GrabPlatformUserDTO createPlatformUser(final CreatePlatformUserDTO create
         return modelMapper.map(platformUserRepository.save(platformUser), GrabPlatformUserDTO.class);
     }
 
-    public GrabPlatformUserDTO updatePlatformUser(final int id, final JsonPatch patch)
+    public GrabPlatformUserDTO updatePlatformUser(final int id, @NotNull final JsonPatch patch)
```

Comment:
I would rename this to be patchPlatformUser as it requires a patch object parameter :)